### ISSUE TITLE
Wrong void return type

### ISSUE THREAD

**Issue by Aerendir** [pipedrive/client-php#59]
Currently all the method of all controllers have the return type hinting set to `void`.

This is wrong, as the real return type is `\Pipedrive\Utils\JsonSerializer`.

Using `void` breaks autocompletion in PHPStorm and breaks static code analyses.

**EXPECTED BEHAVIOR**

Return type hintings in Docblock comments should be `\Pipedrive\Utils\JsonSerializer`.

Example

```diff
    /**
     * Returns all organizations
     *
     * @param  array  $options    Array with all options for search
     * @param integer $options['userId']     (optional) If supplied, only organizations owned by the given user will be
     *                                       returned.
     * @param integer $options['filterId']   (optional) ID of the filter to use
     * @param string  $options['firstChar']  (optional) If supplied, only organizations whose name starts with the
     *                                       specified letter will be returned (case insensitive).
     * @param integer $options['start']      (optional) Pagination start
     * @param integer $options['limit']      (optional) Items shown per page
     * @param string  $options['sort']       (optional) Field names and sorting mode separated by a comma (field_name_1
     *                                       ASC, field_name_2 DESC). Only first-level field keys are supported (no
     *                                       nested keys).
-     * @return void response from the API call
+     * @return \Pipedrive\Utils\JsonSerializer response from the API call
     * @throws APIException Thrown if API call fails
     */
    public function getAllOrganizations(
```

**Comment by branimir93** [comment]
This is duplicate of #4

**Comment by estevanjantsk** [comment]
this issue has been fixed and should be available on [4.0.2](https://github.com/pipedrive/client-php/releases/tag/4.0.2)
